WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListFragment.kt

```diff
@@ -215,6 +206,20 @@ class ActivityLogListFragment : Fragment() {
                 .show(childFragmentManager, ACTIVITY_TYPE_FILTER_TAG)
     }
 
+    private fun updateFilters(uiState: FiltersShown) {
+        with(requireActivity().date_range_picker) {
+            this.text = uiHelpers.getTextOfUiString(requireContext(), uiState.dateRangeLabel)
+            this.isCloseIconVisible = uiState.clearDateRangeFilterClicked != null
+            this.setOnCloseIconClickListener { uiState.clearDateRangeFilterClicked?.invoke() }
+        }
+
+        with(requireActivity().activity_type_filter) {
+            this.text = uiHelpers.getTextOfUiString(requireContext(), uiState.activityTypeLabel)
+            this.isCloseIconVisible = uiState.clearActivityTypeFilterClicked != null
+            this.setOnCloseIconClickListener { uiState.clearActivityTypeFilterClicked?.invoke() }
+        }
+    }
```

Comment:
👍 Fixed in 07479bb4a4b3104b5cd1e286a6f72b124158d7d1

Note: I usually prefer to keep it to emphasise which variables/methods are related to the `with`. However, in this case it doesn't make much sense, since we use it just for value assignment.